### PR TITLE
Foreman 3.2.0 and 3.2.1 release notes

### PR DESCRIPTION
**Pull request by upadhyeammit**
Cherry-pick into:
None

### CODE REVIEW

File: guides/doc-Release_notes/topics/foreman-3.2.0.adoc [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+* Include Tower extra vars feature when calling the API callback - https://projects.theforeman.org/issues/33942[#33942]
+* Add current date macro - https://projects.theforeman.org/issues/33907[#33907]
+* include repo support for puppet 7 in provisioning snippets - https://projects.theforeman.org/issues/32602[#32602]
+
+=== Users, Roles and Permissions
+
+* Infer of permission name for isolated namespace controllers doesnt work - https://projects.theforeman.org/issues/34506[#34506]
+* b"Mail enabled' setting cannot be switched with the hammer user command."Mail enabled setting cannot be switched with the hammer user command." setting cannot be switched with the hammer user command." - https://projects.theforeman.org/issues/34123[#34123]
+
+=== Compute resources - VMware
```
Comment (Lennonka) [inline guides/doc-Release_notes/topics/foreman-3.2.0.adoc:59]: Please, title-style capitalization in all headings

File: guides/doc-Release_notes/topics/foreman-3.2.0.adoc [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+* The login page exposes version of the foreman - https://projects.theforeman.org/issues/33417[#33417]
+
+=== Inventory
+
+* The Submit button on a host form redirects to an invalid page - https://projects.theforeman.org/issues/34572[#34572]
+
+=== Web Interface
+
+* Add pagelet mountpoints for hosts list table - https://projects.theforeman.org/issues/34543[#34543]
+* default variant prop is missing from pagination component - https://projects.theforeman.org/issues/34410[#34410]
```
Comment (Lennonka) [inline guides/doc-Release_notes/topics/foreman-3.2.0.adoc:22]: Please, watch that the first letter is uppercase where it's a normal word - also elsewhere

File: guides/doc-Release_notes/topics/foreman-3.2.0.adoc [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+* Rename enable-puppetlabs-puppetX-repo to enable-puppetofficial-puppetX-repo - https://projects.theforeman.org/issues/34088[#34088]
+* Rename Job Invocation template - https://projects.theforeman.org/issues/34068[#34068]
+* Include Tower extra vars feature when calling the API callback - https://projects.theforeman.org/issues/33942[#33942]
+* Add current date macro - https://projects.theforeman.org/issues/33907[#33907]
+* include repo support for puppet 7 in provisioning snippets - https://projects.theforeman.org/issues/32602[#32602]
+
+=== Users, Roles and Permissions
+
+* Infer of permission name for isolated namespace controllers doesnt work - https://projects.theforeman.org/issues/34506[#34506]
+* b"Mail enabled' setting cannot be switched with the hammer user command."Mail enabled setting cannot be switched with the hammer user command." setting cannot be switched with the hammer user command." - https://projects.theforeman.org/issues/34123[#34123]
```
Comment (Lennonka) [inline guides/doc-Release_notes/topics/foreman-3.2.0.adoc:57]: b? :)

File: guides/doc-Release_notes/topics/foreman-3.2.0.adoc [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+* Grammatical errors with Insecure help text at Host Registration - https://projects.theforeman.org/issues/34250[#34250]
+* Host facts are not uploaded to Foreman when host is registered with global registration - https://projects.theforeman.org/issues/34249[#34249]
+* Fix bool params in global registration template - https://projects.theforeman.org/issues/34206[#34206]
+* Preview of registration and host_init_config templates - https://projects.theforeman.org/issues/34198[#34198]
+* Fix name & path to OS host_init_config template - https://projects.theforeman.org/issues/33979[#33979]
+
+=== Settings
+
+* The Setting defaults are never updated
+ - https://projects.theforeman.org/issues/34323[#34323]
```
Comment (Lennonka) [inline guides/doc-Release_notes/topics/foreman-3.2.0.adoc:125]: Remove linebreak?

File: guides/doc-Release_notes/topics/foreman-3.2.0.adoc [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+* Speed up settings index page - https://projects.theforeman.org/issues/33935[#33935]
+* drop fix_db_cache setting - https://projects.theforeman.org/issues/33906[#33906]
+* Deprecate old setting definitions without \`default_settings\` - https://projects.theforeman.org/issues/33781[#33781]
```
Comment (Lennonka) [inline guides/doc-Release_notes/topics/foreman-3.2.0.adoc:128]: Why is the code fromatting escaped? Also other occurences.

File: guides/doc-Release_notes/topics/foreman-3.2.0.adoc [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+* Add ct/fcct transpiler macro support - https://projects.theforeman.org/issues/31027[#31027]
+
+=== Authentication
+
+* Support ADFS-style JWTs - https://projects.theforeman.org/issues/33055[#33055]
+
+== Installer
+
+* katello-rhsm-consumer script subscription-manager version detection depends on subscription-manager rpm being installed
+ - https://projects.theforeman.org/issues/34406[#34406]
```
Comment (Lennonka) [inline guides/doc-Release_notes/topics/foreman-3.2.0.adoc:233]: Remove linebreak?

**Review by Lennonka — CHANGES_REQUESTED** [review]
I'm assuming that RN for Foreman at the beginning will be added at a later time.

I'm unsure about upstream convention for code formatiing, i.e. whether we want to code-format stuff like method names, config labels and this kind of stuff. It's a lot of work that might not be necessary, but in either case, we should unify this across the whole doc.
I'm leaving it for your consideration.

Plus detailed comments that I would personally prefer to be corrected.

**Comment by upadhyeammit** [comment]
> I'm assuming that RN for Foreman at the beginning will be added at a later time.
> 
> I'm unsure about upstream convention for in-line code formatting, i.e. whether we want to code-format stuff like method/macro names, config labels and this kind of stuff. It's a lot of work that might not be necessary, but in either case, we should unify this across the whole doc. I'm leaving it for your consideration.
> 
> Plus detailed comments that I would personally prefer to be corrected.

I preference will be having quotes around macros or methods. However the Redmine titles will require lots of improvement from each developer and the team. I will discuss on this in upcoming meetings and then we will follow one common format.

For these notes I tried my best to improve titles, if you can spot any more changes then suggest those. And yes I have also updated the Redmine issues, that will help individuals to know the titles were not perfect and next time they will think more on that before creating the issue.

Thanks a lot for review.

File: guides/doc-Release_notes/master.adoc [hunk 7]
```diff
@@ -15,6 +15,9 @@ include::topics/foreman.adoc[leveloffset=+1]
 include::topics/katello.adoc[leveloffset=+1]
 endif::[]
 
+include::topics/foreman-3.2.1.adoc[leveloffset=+1]
+include::topics/foreman-3.2.0.adoc[leveloffset=+1]
```
Comment (maximiliankolb) [inline guides/doc-Release_notes/master.adoc:19]:
```suggestion

include::topics/foreman-3.2.0.adoc[leveloffset=+1]
```

**Comment by maximiliankolb** [comment]
We don't use this downstream, but your PR looks good to me. Any objections @Lennonka ?

**Review by Lennonka — APPROVED** [review]
Ack, looks great

**Pull request merged**